nym-vpn-lib/src/platform/mod.rs

```diff
@@ -209,3 +209,30 @@ async fn get_low_latency_entry_country(
 
     Ok(country)
 }
+
+uniffi::custom_type!(EntryPoint, String);
+uniffi::custom_type!(ExitPoint, String);
```

Comment:
Suggestion: Maybe put this up with the rest?

Reply:
Done!